examples/create-react-app-typescript/package.json

```diff
@@ -16,9 +16,6 @@
     "start": "react-scripts start",
     "build": "react-scripts build"
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
Right now there are errors committing due to eslint, so disable eslinting for the example until we have a chance to audit our lint rules.

Reply:
@scheul93, you were getting this same error earlier this week. Do you agree with the temporary fix to just remove the linting for the example?

Reply:
Seems fine to me. Especially because these examples are more for internal testing purposes